src/types/util/Option.ts

```diff
@@ -0,0 +1 @@
+export type Option<T> = T | null;
```

Comment:
So there is no such type in Typescript itself?

Reply:
Yea it doesn't exist out of the box.

Reply:
Maybe we should make a PR for `Option` to [TypeScript](https://github.com/microsoft/TypeScript) ;) Native support for this would be very nice.